src/server/pegasus_server_impl.cpp

```diff
@@ -2111,6 +2117,19 @@ void pegasus_server_impl::update_server_rocksdb_statistics()
     } else {
         dinfo("_pfc_rdb_block_cache_mem_usage: 0 bytes because block cache is disabled");
     }
+
+    // Update _pfc_rdb_write_limiter_rate_bytes
+    if (_s_rate_limiter) {
+        uint64_t current_total_through = _s_rate_limiter->GetTotalBytesThrough();
+        uint64_t through_rate_per_sec =
+            (current_total_through - _rocksdb_limiter_last_total_through) /
+            kServerStatUpdateTimeSec.count();
+        _pfc_rdb_write_limiter_rate_bytes->set(through_rate_per_sec);
+        _rocksdb_limiter_last_total_through = current_total_through;
+        dinfo_f("_pfc_rdb_write_limiter_rate_bytes: {} bytes", through_rate_per_sec);
+    } else {
+        dinfo_f("_pfc_rdb_write_limiter_rate_bytes: 0 bytes because rate limiter is disabled");
```

Comment:
Since statistics interval has been changed to 10s, so these log seems too duplicate.